install_config/install/prerequisites.adoc

```diff
@@ -214,15 +214,115 @@ The following must be set up in your environment before {product-title} can be
 installed.
 
 [[prereq-dns]]
-
 === DNS
 
+A fully functional DNS environment is a requirement for {product-title} to work
+correctly. Adding entries into the *_/etc/hosts_* file is not enough, because
```

Comment:
Should the above be:

"{product-title} needs a fully functional DNS environment to work correctly."